core/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -17,21 +17,20 @@
 package io.delta.tables
 
 import java.sql.Timestamp
-
 import scala.collection.JavaConverters._
-
 import org.apache.spark.sql.delta._
 import org.apache.spark.sql.delta.actions.Protocol
 import org.apache.spark.sql.delta.catalog.DeltaTableV2
 import org.apache.spark.sql.delta.sources.DeltaSQLConf
 import io.delta.tables.execution._
 import org.apache.hadoop.fs.Path
-
 import org.apache.spark.annotation._
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.TableIdentifier
 import org.apache.spark.sql.types.StructType
 
+import scala.util.Try
```

Comment:
nit: this goes with the other scala library imports

Reply:
Roger